package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "broccoli-babel-transpiler",
-  "version": "5.4.5",
+  "version": "5.5.0",
```

Comment:
5.5.0 is the latest tagged npm version, but master still shows 5.4.5 breaking shrinkwrap